workflow/snakemake_rules/core.smk

```diff
@@ -225,7 +225,7 @@ rule realign:
     input:
         slicedalignment=rules.cut.output.slicedalignment,
         reference=build_dir
-        + "/{a_or_b}/{build_name}/{resolution}/{gene}_reference.fasta",
+        + "/{a_or_b}/{gene}_reference.fasta",
```

Comment:
_noting because I was confused_

Rules specific to processing genes _before_ the `hybrid_align` rule use the `{gene}` wildcard because the genome build needs the G gene alignment in `hybrid_align`, so we needed a separate wildcard from `{build_name}`.